include/kframework/foundry.md

```diff
@@ -1418,6 +1471,9 @@ If the production is matched when no prank is active, it will be ignored.
     rule ( selector ( "expectEmit(bool,bool,bool,bool,address)" )  => 2176505587 )
     rule ( selector ( "sign(uint256,bytes32)" )                    => 3812747940 )
     rule ( selector ( "symbolicStorage(address)" )                 => 769677742  )
+    rule ( selector ( "freshSInt(uint8)" )                         => 625253732  )
```

Comment:
```suggestion
    rule ( selector ( "freshSInt(uint8)" )                         => 978336391  )
```
Probably a copy-paste mistake, `625253732` is the selector for `freshUInt(uint8)` below